vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -48,7 +48,7 @@ function Get-ContainerImages {
     $imagesToPull = @()
 
     switch ($windowsSKU) {
-        '2019' {
+        { '2019', '2019-containerd'} {
             $imagesToPull = @(
```

Comment:
Is the pause image used by containerd as same as that used by docker?

Reply:
Yes, I think these OCI images should be shared container runtimes that implements the OCI Runtime Specification